cpp/src/arrow/dataset/file_orc.cc

```diff
@@ -85,24 +85,20 @@ class OrcScanTask : public ScanTask {
           included_fields.push_back(name);
         }
 
+        std::shared_ptr<RecordBatchReader> recordBatchReader;
```

Comment:
We use `lower_case` for local variables.

Reply:
done